persistence-modules/hibernate-exceptions/src/test/java/com/baeldung/hibernate/exception/transientobject/HibernateTransientObjectUnitTest.java

```diff
@@ -0,0 +1,88 @@
+package com.baeldung.hibernate.exception.transientobject;
+
+import com.baeldung.hibernate.exception.transientobject.entity.Address;
+import com.baeldung.hibernate.exception.transientobject.entity.Department;
+import com.baeldung.hibernate.exception.transientobject.entity.Author;
+import com.baeldung.hibernate.exception.transientobject.entity.Book;
+import com.baeldung.hibernate.exception.transientobject.entity.Employee;
+import com.baeldung.hibernate.exception.transientobject.entity.User;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class HibernateTransientObjectUnitTest {
+
+    private static SessionFactory sessionFactory;
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @BeforeClass
+    public static void init() {
+        sessionFactory = HibernateUtil.getSessionFactory();
+    }
+
+    @Test
+    public void whenSaveEntitiesWithOneToOneAssociation_thenSuccess() {
+
```

Comment:
As per guidelines, avoid blank lines immediately after an opening brace. All the test methods have this.

Reply:
Done.